### DISCUSSION TITLE
Release 2.20?

### DISCUSSION THREAD

**Discussion by ooxi** [cc65/cc65#1948] [General]
What's stopping us from releasing 2.20? The version 2.19 carried by most distributions is lacking significantly behind current master with features like `_Static_assert`.

**Comment by polluks** [comment]
@ooxi Please take care of #1667.

**Comment by mrdudz** [comment]
srsly, what exactly do you think has to be taken care of there? (why not look into #1856 instead? *sigh*)

As for making a release, i tried to collect some things [here](https://github.com/cc65/wiki/wiki/Before-the-next-release). I want to atleast have all buildsystem and testing things done first, aswell as generating/updating docs and website automatically.

**Comment by ooxi** [comment]
@polluks #1667 is resolved, isn't it?

Thanks for the feedback @mrdudz

**Comment by lydon42** [comment]
I am one of the MEGA65 maintainers and we are still sticking to the V2.19 tag. ~~Recently I stumbled over some problems (sorry, forgot what the exact problem was), which could be solved by using a build of the HEAD of master.~~ _Edit: my problem was with a prebuild debian package used in our docker, sorry, I misremembered!_

What is your current recommendation? Stick to V2.19? Use HEAD of master? It seems the 2.20 discussion here has dried up, and both referenced issues seem to be closed...

Edit: sorry for bumping up an old discussion, I should have researched the problem before making to much noise! :)

**Comment by mrdudz** [comment]
I recommend to use head, always. Also see above. A release will happen when at least the buildsystem is completely fixed as lined out in the wiki, and at least all regressions over 2.19 are fixed. And as always, if you want to speed up the process, please help. I dont see it happening anytime soon right now (unfortunately one of the most active contributors, @acqn, is MIA since december)

**Reply by rofl0r** [reply]
i guess for distro folks it would be helpful to have a tag, at least. maybe something like 2.20-beta-20230429, so they dont have to pin their recipes to some random sha1 hash without any indication what the version is.

**Reply by lydon42** [reply]
One confusing point of the V2.19 tag is that the resulting binary tells you that it is `cc65 V2.18 - Git 555282497`, as the version was bumped near the end of 2020. I don't know if this is the case with all distros or if some of them patched the version on their own. Making a new release would also clean that up.
